src/App.tsx

```diff
@@ -1,42 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
-export const tabs = [
+import { Tab } from './interface/interface';
+
+export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const tabSelected = (tab: Tab) => {
+    setSelectedTabId(tab.id);
+  };
```

Comment:
```suggestion
  const tabSelected = (id: string) => {
    setSelectedTabId(id);
  };
```

Reply:
![image](https://user-images.githubusercontent.com/98056446/230767473-2248e23f-ff7e-45e3-a3ec-c7cf81fe9f18.png)
I left a comment about this in previous messages. If I pass only id, one of the tests will be fail